.travis.yml

```diff
@@ -6,24 +6,27 @@ php:
   - 5.5
   - 5.6
   - 7.0
-  - hhvm
-  - hhvm-nightly
+  - 7.1
+  - 7.2
+  - 7.3
+  - 7.4snapshot
+  - nightly
 
 matrix:
   allow_failures:
-    - php: hhvm
-    - php: hhvm-nightly
+    - php: 7.4snapshot
+    - php: nightly
 
 before_script:
   - git fetch --unshallow --tags
   - composer install --prefer-source
-  - composer global require 'kherge/box=~2.4' --prefer-source
+  - curl -LSs https://box-project.github.io/box2/installer.php | php
   - wget https://scrutinizer-ci.com/ocular.phar
```

Comment:
this is fixed already, using composer still but using the travis (or composer) new path for global, ~/.config/composer